rules-reviewed/eap6/weblogic/weblogic.windup.xml

```diff
@@ -35,6 +38,8 @@
                       6 Tutorial](https://docs.oracle.com/javaee/6/tutorial/doc/bnboy.html).
                       ]]>
                     </message>
+                    <link href="https://docs.oracle.com/javaee/6/tutorial/doc/bnboy.html" title="EJB Timer Service" />
```

Comment:
there is already a link in hint message, the same URL but different title ? it looks weird in the report
![screenshot from 2017-06-16 12-52-54](https://user-images.githubusercontent.com/177257/27223927-e8b1583a-5292-11e7-8aeb-be281cb59df3.png)